esp/src/src-react/components/Workflows.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from "react";
+import { CommandBar, ContextualMenuItemType, ICommandBarItemProps } from "@fluentui/react";
+import { useConst } from "@fluentui/react-hooks";
+import { AlphaNumSortMemory } from "src/Memory";
+import * as Observable from "dojo/store/Observable";
+import nlsHPCC from "src/nlsHPCC";
+import { useWorkunitWorkflows } from "../hooks/Workunit";
+import { HolyGrail } from "../layouts/HolyGrail";
+import { ShortVerticalDivider } from "./Common";
+import { DojoGrid } from "./DojoGrid";
+
+interface WorkflowsProps {
+    wuid: string;
```

Comment:
We usually use `Wuid` rather than `wuid`, yeah?

Reply:
lowercase (camel case) for properties - "Wuid" is typically the value when returned from ESP.